gtsfm/loader/hilti_loader.py

```diff
@@ -90,22 +93,26 @@ def __init__(
         if self._max_length is not None:
             self.num_rig_poses = min(self.num_rig_poses, self._max_length)
 
+        # Read GT poses from odom file:
+        self._fastlio_poses = self.__read_fastlio_poses()
+
         # Read the constraints from the lidar/constraints file
         all_constraints: Dict[Tuple[int, int], Constraint] = self.__load_constraints()
         filtered_constraints: Dict[Tuple[int, int], Constraint] = self._filter_outlier_constraints(all_constraints)
         self._constraints: Dict[Tuple[int, int], Constraint] = self._update_stationary_constraints(filtered_constraints)
 
-        logger.info("Number of constraints: %d", len(self._constraints))
+        logger.info("[hilti_loader] Number of constraints: %d", len(self._constraints))
```

Comment:
Nit: We have configured the logger to use the file name when it logs, so tag `[hilti_loader]` is redundant.

Reply:
Oh, I see! I have been changing these all over ;-)